account_mtd/models/mtd_hello_world.py

```diff
@@ -0,0 +1,406 @@
+                else:
+                    # THe request made was within 10 mins so the user has to wait.
+                    test = True
+                    raise exceptions.Warning(
+                        "An authorisation request is already in process!!!\n " +
+                        "Please try again later"
+                    )
+            else:
+                self._logger.info(
+                    "Connection button Clicked - endpoint name {}, ".format(self.name) +
+                    "We have access token and refresh token"
+                )
+                version = self._json_command('version')
+                return version
+        else:
+            raise exceptions.Warning("Could not connect to HMRC! \nThis is not a valid end point")
+    
+    def _json_command(self, command, timeout=3, record_id=None):
+        # this will determine where we have come from
+        # if there is no record_id then we know we already had valid record to gain all the information for hello user
+        self._logger.info(
+            "_json_command - has record_id been provided?:- {}".format(record_id)
+        )
+        if record_id:
+            self = self.env['mtd.hello_world'].search([('id', '=', record_id)])
+            self._logger.info(
+                "_json_command - we need to find the record and assign it to self"
+            )
+        # search the token table to see if we have access token
+        token_record = self.env['mtd.api_tokens'].search([('api_id', '=', self.api_name.id)])
+        access_token = token_record.access_token if token_record else ""
+        # may not newed next line of code will need to look into this further while testing.
+        # refresh_token = token_record.refresh_token if token_record else ""
+        
+        header_items = {"Accept": "application/vnd.hmrc.1.0+json"}
+        if self.which_button_type_clicked == "application":
+            header_items["authorization"] = ("Bearer "+self.hmrc_configuration.server_token)
+        elif self.which_button_type_clicked == "user":
+            # need to first check if the user has any accessToken and refreshtoken
+            # If not we need to proceed to the first and second step and then come to this step.
+            header_items["authorization"] = ("Bearer "+str(access_token))
+
+        hmrc_connection_url = "{}{}".format(self.hmrc_configuration.hmrc_url, self.path)
+        self._logger.info(
+            "_json_command - hmrc connection url:- {}, ".format(hmrc_connection_url) +
+            "headers:- {}".format(header_items)
+        )
+        req = requests.get(hmrc_connection_url, timeout=3, headers=header_items)
+        response_token = json.loads(req.text)
+        self._logger.info(
+            "_json_command - received respponse of the request:- {}, ".format(req) +
+            "and its text:- {}".format(response_token)
+        )
+        if req.ok:
+            construct_text = (
```

Comment:
`error_message` a better name. 

Consider using `+` at start of the lines instead of end, or even better using a single format string with named parameters.